packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -200,6 +224,19 @@ function getTxIdFromTransaction(tx: Transfer) {
   return tx.l2TxHash ?? tx.transactionHash
 }
 
+function getCacheKeyFromTransaction(
+  tx: Transaction | MergedTransaction | TeleportFromSubgraph | Withdrawal
+) {
+  return `${tx.parentChainId}-${getTxIdFromTransaction(tx)?.toLowerCase()}`
```

Comment:
We should probably do something like this so we don't end up with `{tx.parentChainId}-undefined`.
```
const txId = getTxIdFromTransaction(tx)

if (!txId) {
   return undefined
}

return `${tx.parentChainId}-${txId.toLowerCase()}`
```